package exceptions.testthrows;

public class C {
	public void f3(){
		
	}
}
